carbon-p2-plugin/src/main/java/org/wso2/maven/p2/RepositoryGenMojo.java

```diff
@@ -291,8 +292,9 @@ private void copyBundleArtifacts()throws MojoExecutionException {
                 .hasNext();) {
             BundleArtifact bundleArtifact = (BundleArtifact) iterator.next();
             try {
-            	File file = bundleArtifact.getArtifact().getFile();
-                FileManagementUtil.copy(file, new File(pluginsDir,file.getName()));
+            	//This will be fixed in a different jira
```

Comment:
The RepositoryGenMojo in this PR is the direct migration from maven-tools repo to carbon-maven-plugins repo with slight alterations. This is removed after refactoring.